sdk/confidentialledger/Azure.ResourceManager.ConfidentialLedger/api/Azure.ResourceManager.ConfidentialLedger.netstandard2.0.cs

```diff
@@ -171,4 +218,67 @@ public partial class ConfidentialLedgerProperties
         public static bool operator !=(Azure.ResourceManager.ConfidentialLedger.Models.ConfidentialLedgerType left, Azure.ResourceManager.ConfidentialLedger.Models.ConfidentialLedgerType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class DeploymentType
+    {
+        public DeploymentType() { }
+        public System.Uri AppSourceUri { get { throw null; } set { } }
+        public Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime? LanguageRuntime { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct LanguageRuntime : System.IEquatable<Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime>
```

Comment:
The type name is too generic, use `prepend-rp-prefix` to specify it.

Reply:
Done, Thanks!